src/udev/probes.d

```diff
@@ -0,0 +1,11 @@
+provider udev {
+        probe worker_spawned(const char*, const char*, int); /* (sysname, action, pid) */
```

Comment:
hmm, shouldn't thee comments be inside the parameters list? i.e. is that really how this should be done that you put the args nameless in there, and add the arg names only as comment?

Reply:
Well, I based this of off "official" documentation.

https://sourceware.org/systemtap/wiki/AddingUserSpaceProbingToApps